apps/arweave/src/ar_http_iface_middleware.erl

```diff
@@ -2008,21 +2008,24 @@ handle_get_chunk(OffsetBinary, Req, Encoding) ->
 								{Packing, ok};
 							{{true, _}, _StoreID} ->
 								{ok, Config} = application:get_env(arweave, config),
-								case lists:member(pack_served_chunks, Config#config.enable) of
-									false ->
-										{none, {reply, {404, #{}, <<>>, Req}}};
+								IsPackServedChunks = lists:member(pack_served_chunks, Config#config.enable),
+								Peer = ar_http_util:arweave_peer(Req),
+								IsLocalPeerAddr = lists:member(Peer, Config#config.local_peers),
+
+								case IsPackServedChunks orelse IsLocalPeerAddr of
```

Comment:
Should this be `andalso`? I forget where we landed

Like I could see it working like this:
- server sets `pack_served_chunks`, in which case it only repacks chunks for members of its `local_peers`
- client sets `request_packed_chunks` and `sync_from_local_peers_only` in which case it requests packed chunks from its `local_peers` and does not validate it them before storing.

Reply:
We decided that we pack served chunks for local peers even when the option is not set.

Reply:
I think we should always require the option to be set. Unless you and Lev discussed otherwise?

i.e. there are a lot of scenarios where local_peers are set for other reasons (e.g. reduce rate limiting between CM cluster peers) and in those cases we probably don't want to also pack served chunks unless it's desired (e.g. we wouldn't want a CM Exit Node to get bogged down repacking accidentally)

Reply:
We discussed this few times:
- CM exit node will unlikely have any data at all;
- local_peers are considered trusted and therefore any requested packing is served.

Reply:
Just chatted on slack. Current approach as implemented works! clients will only request packed chunks if they have `fetch_packed_chunks` set, but if they do request a packed chunk, the local_peer will repack.